### PR TITLE
Fix: If filter hits on MyPlan page, filter button is shown

### LINKED ISSUE #397: If filter hits on MyPlan page, filter button is shown
**Overview (Required)**
- on MyPlan page set filter and some favorite session hits then filter button appear. (and session list too)

**Steps To Reproduce**

1. Add some sessions to favorite
2. go to MyPlan page
3. show filters and set some filter to be hit favorite session
4. then it is appeared session list and filter button

**Expected behavior**
same as Day1/Day2 page.
(on Day1/Day2 page, this issue dose not occur)

**Screenshots**

myplans_filter_bug

**Env:**
 - Device: Nexus6P
 - OS: 8.1.1 (Android 10 too)
 - master

### PR DESCRIPTION
**Pull request by t-kashima**
## Issue
- close #397

## Overview (Required)

- I changed the conditions for displaying the filter button.
- The filter button is displayed when (has filtered sessions || unfiltered) && close the filter.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
Thanks!
Probably when there are no favorite sessions, the image should be displayed, so the button should not be displayed. 🖼

**Comment by t-kashima** [comment]
Thank you for reviewing!
I changed to not show the filter button when there is no session. 
And I found the filter button is not displayed when there are sessions but no filtered sessions and fixed it.

Please review again 🙏

File: feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetFavoriteSessionsFragment.kt [hunk 1]
```diff
@@ -80,19 +80,7 @@ class BottomSheetFavoriteSessionsFragment : DaggerFragment() {
...
-                    false
-            }
-            binding.startFilter.visibility = when (uiModel.expandFilterState) {
-                ExpandFilterState.EXPANDED, ExpandFilterState.CHANGING ->
-                    View.VISIBLE
-                else ->
-                    View.INVISIBLE
-            }
-            binding.expandLess.isVisible = when (uiModel.expandFilterState) {
+            binding.isCollapsed = when (uiModel.expandFilterState) {
```
Comment (takahirom) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetFavoriteSessionsFragment.kt]: 👏

**Comment by t-kashima** [comment]
Thank you for the review😊

**Pull request merged**